test/DssVestTopUp.test.ts

```diff
@@ -157,10 +157,9 @@ describe("DssVestTopUp", function () {
       });
 
       it("should vest accrued tokens", async function () {
-        await topUp.refundUpkeep();
-        const vested = await token.balanceOf(topUp.address);
-
-        expect(vested).to.eq(maxDepositAmt);
+        await expect(topUp.refundUpkeep())
+          .to.emit(topUp, "VestedTokensWithdrawn")
+          .withArgs(vestedTokens);
```

Comment:
Please use `dssVest.unpaid()` like in the other tests, otherwise it's assumed all tokens are accrued.